presto-hive-hadoop2/bin/start_hive.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+. ${BASH_SOURCE%/*}/common.sh
```

Comment:
```suggestion
. "${BASH_SOURCE%/*}/common.sh"
```

Reply:
I copied this from `run_hive_tests.sh` which also be updated